lib/helpers/make-replacements.js

```diff
@@ -29,6 +29,10 @@ module.exports = function makeReplacements(contents, from, to, file) {
   //Make replacements
   from.forEach((item, i) => {
 
+    if (typeof item === 'function') {
```

Comment:
Can you add a comment above this block to indicate if a function is given it's called with the file name?